### PR TITLE
New Icon: RTL variation of list-ordered

### PR DESCRIPTION
**Pull request by iamthomasbishop**
Adds a new icon for cases where we're showing the ordered list icon in the context of RTL.

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/9478#issuecomment-394774800

Examples:

screenshot 2018-06-05 12 13 53

screenshot 2018-06-05 12 14 18

### CODE REVIEW

**Comment by folletto** [comment]
I think it's good to go! 👍

File: sources/svg/gridicons-list-ordered-rtl.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><rect x="0" fill="none" width="24" height="24"/>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (folletto) [inline sources/svg/gridicons-list-ordered-rtl.svg]: Do some cleanup to the source before final build & merge. :) 
Details on how to do it are here: https://github.com/Automattic/gridicons/wiki/Guidelines-for-SVG-Sources

**Comment by iamthomasbishop** [comment]
@folletto Can you please give this a final look-over and merge?

**Comment by folletto** [comment]
I'm not 100% sure about the translate command, and if it could be removed in some way, but it works, and the minified version doesn't seem to have it, so I'd say it's good to go.

**Pull request merged**